src/auth/auth.proto

```diff
@@ -437,6 +454,8 @@ service API {
 
   rpc GetConfiguration(GetConfigurationRequest) returns (GetConfigurationResponse) {}
   rpc SetConfiguration(SetConfigurationRequest) returns (SetConfigurationResponse) {}
+  rpc AddScopesToConfiguration(AddScopesToConfigurationRequest) returns (AddScopesToConfigurationResponse) {}
+  rpc RemoveScopesFromConfiguration(RemoveScopesFromConfigurationRequest) returns (RemoveScopesFromConfigurationResponse) {}
```

Comment:
These RPCs seem to overlap with the functionality in `GetConfiguration`?

Reply:
Yeah they use similar implementation logic to GetConfiguration and SetConfiguration.

Does having multiple ways to modify Configuration values make for an unclear interface? I could also make these helper functions in the License module